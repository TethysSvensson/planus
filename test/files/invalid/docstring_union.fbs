//! Okay
/// Okay
//! Bad1
/// Okay
union /// Bad2
/// Bad3
Foo /// Bad4
/// Bad5
{ /// Bad6
  //! Bad7
  /// Okay
  x /// Bad8
  /// Bad9
  : /// Bad10
  /// Bad11
  Bar /// Bad12
  /// Bad13
  , /// Bad14
  //! Bad15
  /// Okay
  Baz /// Bad16
/// Bad17
} /// Bad18
//! Bad19


table Bar {}
table Baz {}
