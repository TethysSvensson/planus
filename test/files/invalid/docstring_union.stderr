error: Inner doc comments (those starting with `//!`) are only allowed at the beginning of the file and are used to document the namespace. If you meant to write a normal doc comment, those start with `///`.
  ┌─ test/files/invalid/docstring_union.fbs:3:1
  │
3 │ //! Bad1
  │ ^^^^^^^^

error: Doc comments are not meaningful here.
  ┌─ test/files/invalid/docstring_union.fbs:5:7
  │
5 │ union /// Bad2
  │       ^^^^^^^^

error: Doc comments are not meaningful here.
  ┌─ test/files/invalid/docstring_union.fbs:6:1
  │
6 │ /// Bad3
  │ ^^^^^^^^

error: Doc comments are not meaningful here.
  ┌─ test/files/invalid/docstring_union.fbs:7:5
  │
7 │ Foo /// Bad4
  │     ^^^^^^^^

error: Doc comments are not meaningful here.
  ┌─ test/files/invalid/docstring_union.fbs:8:1
  │
8 │ /// Bad5
  │ ^^^^^^^^

error: Doc comments are not meaningful here.
  ┌─ test/files/invalid/docstring_union.fbs:9:3
  │
9 │ { /// Bad6
  │   ^^^^^^^^

error: Doc comments are not meaningful here.
   ┌─ test/files/invalid/docstring_union.fbs:22:1
   │
22 │ /// Bad17
   │ ^^^^^^^^^

error: Doc comments are not meaningful here.
   ┌─ test/files/invalid/docstring_union.fbs:23:3
   │
23 │ } /// Bad18
   │   ^^^^^^^^^

error: Inner doc comments (those starting with `//!`) are only allowed at the beginning of the file and are used to document the namespace. If you meant to write a normal doc comment, those start with `///`.
   ┌─ test/files/invalid/docstring_union.fbs:10:3
   │
10 │   //! Bad7
   │   ^^^^^^^^

error: Doc comments are not meaningful here.
   ┌─ test/files/invalid/docstring_union.fbs:12:5
   │
12 │   x /// Bad8
   │     ^^^^^^^^

error: Doc comments are not meaningful here.
   ┌─ test/files/invalid/docstring_union.fbs:13:3
   │
13 │   /// Bad9
   │   ^^^^^^^^

error: Doc comments are not meaningful here.
   ┌─ test/files/invalid/docstring_union.fbs:14:5
   │
14 │   : /// Bad10
   │     ^^^^^^^^^

error: Doc comments are not meaningful here.
   ┌─ test/files/invalid/docstring_union.fbs:15:3
   │
15 │   /// Bad11
   │   ^^^^^^^^^

error: Doc comments are not meaningful here.
   ┌─ test/files/invalid/docstring_union.fbs:16:7
   │
16 │   Bar /// Bad12
   │       ^^^^^^^^^

error: Doc comments are not meaningful here.
   ┌─ test/files/invalid/docstring_union.fbs:17:3
   │
17 │   /// Bad13
   │   ^^^^^^^^^

error: Doc comments are not meaningful here.
   ┌─ test/files/invalid/docstring_union.fbs:18:5
   │
18 │   , /// Bad14
   │     ^^^^^^^^^

error: Inner doc comments (those starting with `//!`) are only allowed at the beginning of the file and are used to document the namespace. If you meant to write a normal doc comment, those start with `///`.
   ┌─ test/files/invalid/docstring_union.fbs:19:3
   │
19 │   //! Bad15
   │   ^^^^^^^^^

error: Doc comments are not meaningful here.
   ┌─ test/files/invalid/docstring_union.fbs:21:7
   │
21 │   Baz /// Bad16
   │       ^^^^^^^^^

error: Inner doc comments (those starting with `//!`) are only allowed at the beginning of the file and are used to document the namespace. If you meant to write a normal doc comment, those start with `///`.
   ┌─ test/files/invalid/docstring_union.fbs:24:1
   │
24 │ //! Bad19
   │ ^^^^^^^^^

